### ISSUE TITLE
SymPy tests fail in IPython 5b3 

### ISSUE THREAD

**Issue by asmeurer** [ipython/ipython#9598]
Run 

```
./bin/test ipython
```

from the SymPy repo with the IPython 5b3 beta installed. This is the test failure.

```
______________________________________________________________________________________________________________________________________________________________
_________________________________________ sympy/interactive/tests/test_ipythonprinting.py:test_print_builtin_option __________________________________________
  File "/users/aaronmeurer/documents/python/sympy/sympy/sympy/interactive/tests/test_ipythonprinting.py", line 66, in test_print_builtin_option
    raises(KeyError, lambda: app.user_ns['a'][0]['text/latex'])
  File "/Users/aaronmeurer/Documents/Python/sympy/sympy/sympy/utilities/pytest.py", line 81, in raises
    raise AssertionError("DID NOT RAISE")
AssertionError: DID NOT RAISE
```

**Comment by minrk** [comment]
On the case!

**Comment by minrk** [comment]
Fixed by #9601
